### PR TITLE
implicit connect & bug fixes

### PR DESCRIPTION
**Pull request by Ekwav**

### CODE REVIEW

File: src/main/java/de/torui/coflsky/Config.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package de.torui.coflsky;
 
 public class Config {
-    public static final int KeepFlipsForSeconds = 50;
+    public static final int KeepFlipsForSeconds = 42;
```
Comment (ToruiDev) [inline src/main/java/de/torui/coflsky/Config.java:4]: may i ask why this specific change?

File: src/main/java/de/torui/coflsky/FlipperChatCommand.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		return "Shorthand for /cofl chat";
+	}
+	
+	@Override
+	public void processCommand(ICommandSender sender, String[] args) throws CommandException {
+		new Thread(()->{
+            String[] newArgs = new String[args.length +1];
+			System.arraycopy(args, 0, newArgs, 1, args.length);
+            newArgs[0] = "chat";
+			CommandNotRecognized(newArgs, sender);
```
Comment (ToruiDev) [inline src/main/java/de/torui/coflsky/FlipperChatCommand.java:49]: i feel like this is abusing this function.
should it perhaps be renamed?
(just a suggestion, does not block merge)

File: src/main/java/de/torui/coflsky/Config.java (hunk 1 above)
Comment (Ekwav) [reply src/main/java/de/torui/coflsky/Config.java:4]: New auctions for flips are available every 60 seconds and it commonly takes more than 5 seconds to process them.
However People wanted to hold the key down earlier than 5 seconds causing old flips to be opened

File: src/main/java/de/torui/coflsky/FlipperChatCommand.java (hunk 2 above)
Comment (Ekwav) [reply src/main/java/de/torui/coflsky/FlipperChatCommand.java]: Sounds reasonable, renamed it

**Pull request merged**